index.css

```diff
@@ -340,6 +339,146 @@ q:before, q:after{ content: "";}
     margin-top: 6px;
 }
 
+/* shop container */
+
+.shop-container {
+    position: relative;
+    text-align: center;
+}
+
+.shop-heading {
+    margin: 0 auto;
+    width: 218px;
+    padding-bottom: 45px;
+}
+
+.heading__subtitle {
+    opacity: 0.9;
+    color: #5da8ef;
+    font-size: 9px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 6px;
+    text-align: center;
+    display: block;
+}
+
+.heading__title {
+    color: #333333;
+    font-size: 42px;
+    font-weight: 700;
+    line-height: 49px;
+    margin:0;
+}
+
+.shop-grid {
+    padding-bottom: 72px;
+}
+
+.shop-grid__col-01,
+.shop-grid__col-02,
+.shop-grid__col-03 {
+    position: relative;
+    margin-right: 3.3%;
+    width: 31.1%;
+    height: 442px;
+}
+
+.shop-grid__col-03 {
+    margin-right: 0;
+}
+
+.catalog__surf-image {
+    -webkit-filter: drop-shadow(5px 5px 5px #d1d3e1);
+    filter: drop-shadow(11px 5px 4px #d1d3e1);
+    position: absolute;
+    bottom: 28px;
+    left: 66px;
+}
+
+.shop-grid__col-01:after,
+.shop-grid__col-02:after,
+.shop-grid__col-03:after {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 250px;
+    height: 320px;
+    background-color: #e8e8e8;
+    z-index: -1;
+    bottom: 0;
+    left: 0;
+}
+
+.catalog__content {
+    position: absolute;
+    bottom: 107px;
+    right: -4px;
+    text-align: left;
+}
+
+.content__subtitle {
+    color: #7a7a7a;
+    font-size: 12px;
+    font-weight: 300;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    display: block;
+}
+
+.content__title {
+    color: #333333;
+    font-size: 26px;
+    font-weight: 400;
+    line-height: 28px;
+    margin: 0;
+}
+
+.content__price {
+    color: #ce60a8;
+    font-size: 20px;
+    font-weight: 300;
+    display: block;
+    margin-left: 70px;
+    margin-top: 32px;
+}
+
+.content__buy {
+    color: #333333;
+    font-size: 9px;
+    font-weight: 700;
+    text-transform: uppercase;
+    margin-left: 70px;
+    display: inline-block;
+    letter-spacing: 5px;
+    margin-top: 5px;
+    text-decoration: none;
+}
+
+.content__buy:hover {
+    color: #5da8ef;
+}
+
+.shop-container button {
+    position: relative;
+    border-color: transparent;
+    background-color: transparent;
+}
+
+.shop-controls__previous-arrow {
+    position: absolute;
+    bottom: 250px;
+    left: -599px;
```

Comment:
The `left` looks like a magic (positioning) number to me :-)

Move the `.shop-controls__previous-arrow` properties (absolute, left, bottom) to the `<button>` element. Then - your issue will be resolved and you can position it easily.